roles/launch-nomad-jobs/templates/souptik.nomad.j2

```diff
@@ -36,11 +36,19 @@ job "souptik_job" {
       }
 
       env = {
-        WORDPRESS_DB_HOST     = "{{ DatabaseHost }}"
-        WORDPRESS_DB_USER     = "{{ SouptikDatabaseUser }}"
-        WORDPRESS_DB_PASSWORD = "{{ SouptikDatabasePassword }}"
-        WORDPRESS_DB_NAME     = "{{ SouptikDatabaseName }}"
-        WORDPRESS_DEBUG       = "{{ SouptikDebugLog }}"
+        WORDPRESS_DB_HOST      = "{{ DatabaseHost }}"
+        WORDPRESS_DB_USER      = "{{ SouptikDatabaseUser }}"
+        WORDPRESS_DB_PASSWORD  = "{{ SouptikDatabasePassword }}"
+        WORDPRESS_DB_NAME      = "{{ SouptikDatabaseName }}"
+        WORDPRESS_DEBUG        = "{{ SouptikDebugLog }}"
+        WORDPRESS_CONFIG_EXTRA = "
+        define( 'MULTISITE', true );
+        define( 'SUBDOMAIN_INSTALL', true );
+        define( 'DOMAIN_CURRENT_SITE', 'souptik.taptappers.club' );
```

Comment:
You can use `souptik.{{ BaseDomain }}` instead. That'll abstract out the domain name. `BaseDomain` variable is already available.